test/test_threaded.jl

```diff
@@ -108,6 +125,16 @@ Trixi.mpi_isroot() && isdir(outdir) && rm(outdir, recursive=true)
       )
     end
 
+    @trixi_testset "elixir_euler_curved.jl with threaded time integration" begin
+      @test_broken false
+      # TODO: This is cirrently broken and needs to be fixed upstream
```

Comment:
```suggestion
      # TODO: This is currently broken and needs to be fixed upstream
```
Also, maybe add a link to an issue?

Reply:
59c6bf0a3 :+1: